packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputFiatCrypto/CoinmarketFormInputCryptoAmount.tsx

```diff
@@ -74,7 +71,7 @@ export const CoinmarketFormInputCryptoAmount = <TFieldValues extends CoinmarketA
     const networkSymbol = cryptoSelect?.value && cryptoIdToCoinSymbol(cryptoSelect?.value);
     const decimals = getCoinmarketNetworkDecimals({
         sendCryptoSelect: !isCoinmarketBuyContext(context)
-            ? context.getValues()[FORM_SEND_CRYPTO_CURRENCY_SELECT]
+            ? (cryptoSelect as CoinmarketAccountOptionsGroupOptionProps | undefined)
```

Comment:
I prefer to get already typed value from context - updated code over force typing. What do you think?

Reply:
It will be more readable when the exact property is passed.